iot/api-client/manager/resources/rsa_private.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Did you really mean to publish this?

Reply:
Yes, it's used after the device is registered to connect and test the "http device" samples.  The authorization credential is revoked during tear-down of the tests.

Reply:
At any rate, this file is needed to connect and providing a dummy / static one is probably simpler than generating them with code or shelling out to `openssl` in the tests.

Reply:
Sg - It won't surprise me for this to trigger some scanners, in which case, I'll ask you to encrypt a new one and we can decrypt when we test.